src/tennisSimulator/Serve.java

```diff
@@ -0,0 +1,46 @@
+package tennisSimulator;
+
+public class Serve {
+
+    private final Player firstPlayer;
+    private final Player secondPlayer;
+    private final Double SERVING_FACTOR = 0.2;
+    private final Integer RATING_DIFFERENCE;
+
+    public Serve(Player firstPlayer, Player secondPlayer) {
+        this.firstPlayer = firstPlayer;
+        this.secondPlayer = secondPlayer;
+        RATING_DIFFERENCE = firstPlayer.getAttributes().getRating() - secondPlayer.getAttributes().getRating();
+    }
+
+    public void defineFirstServer() {
+        if (Math.random() >= 0.5) {
+            firstPlayer.getMatchStats().setServing(true);
+        } else {
+            secondPlayer.getMatchStats().setServing(true);
+        }
+    }
+
+    public void changeServer() {
+        if (firstPlayer.getMatchStats().isServing()) {
+            firstPlayer.getMatchStats().setServing(false);
```

Comment:
I would image that you would call: firstPlayer.setServing(false) and it would delegate to match stats that job.